src/main/java/FTPServer.java

```diff
@@ -267,7 +267,6 @@ private void readRequest(RequestMessage message) throws IOException {
 			byte [] fileBytes = resourceManager.readFileToBytes(message.getFileName());
 			sendDataBlock(fileBytes);
 		} catch (FileNotFoundException e) {
-			e.printStackTrace();
```

Comment:
:fire: this line because it looks like an uncaught error when we print this stack-trace even though it doesn't crash the server.